tests/end_to_end/CairoPrecompiles/test_dual_vm_token.py

```diff
@@ -36,6 +44,29 @@ async def dual_vm_token(kakarot, starknet_token, owner):
     return dual_vm_token
 
 
+@pytest_asyncio.fixture(scope="function", autouse=True)
+async def fund_owner(owner, starknet_token, max_fee):
```

Comment:
I dont understand the need for this fixture

Reply:
this one is required if we use the deployed DualVMTokens in the tests. Can be removed if I keep using test-tokens for the tests instead of "official" tokens.